src/Microsoft.Extensions.Options/OptionsServiceCollectionExtensions.cs

```diff
@@ -37,7 +39,18 @@ public static IServiceCollection AddOptions(this IServiceCollection services)
         /// <param name="services">The <see cref="IServiceCollection"/> to add the services to.</param>
         /// <param name="configureOptions">The action used to configure the options.</param>
         /// <returns>The <see cref="IServiceCollection"/> so that additional calls can be chained.</returns>
-        public static IServiceCollection Configure<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions)
+        public static IServiceCollection Configure<TOptions>(this IServiceCollection services, Action<TOptions> configureOptions) where TOptions : class
+            => services.Configure(string.Empty, configureOptions);
```

Comment:
>string.Empty [](start = 34, length = 12)

Looks like we're saying un-named (legacy) options is just a named options where the name is the empty string. We should consider if this is the right "default name" to use.